src/main/resources/db/migration/V2024_03_22_21_33__insertDataIntoReviews.sql

```diff
@@ -1,13 +1,13 @@
 INSERT INTO reviews (status, employee_id, created_at) VALUES
 ('DONE', 1, '2022-01-01 12:00:00'),
-('CANCELLED', 2, '2022-01-02 10:30:00'),
```

Comment:
I fixed the migration file because otherwise you can get the combination of reviewStatus == CANCELLED and employee.isActive == true after running the ReviewService.checkAndUpdateReviews() method. In my opinion, this does not correspond to the application logic